demisto_sdk/commands/common/tools.py

```diff
@@ -208,24 +209,29 @@ def get_remote_file(
     tag = tag.replace('origin/', '').replace('demisto/', '')
 
     github_path = urljoin(github_config.CONTENT_GITHUB_LINK, tag, full_file_path)
-
+    github_request = partial(requests.get, github_path, verify=False, timeout=10)
     try:
-        headers = {}
-        if is_external_repository():
+        external_repo = is_external_repository()
+        if external_repo:
             githhub_config = GithubContentConfig()
             if githhub_config.Credentials.TOKEN:
-                headers = {
+                res = requests.get(github_path, verify=False, timeout=10, headers={
                     'Authorization': f"Bearer {githhub_config.Credentials.TOKEN}",
                     'Accept': f'application/vnd.github.VERSION.raw',
-                }
+                })  # Sometime we need headers
+                if not res.ok:  # sometime we need param token
+                    res = github_request(params={'token': githhub_config.Credentials.TOKEN})
+                res.raise_for_status()
             else:
-                print_color(
+                raise DemistoException(
```

Comment:
this will stop the command run - do we want that?

Reply:
It'll not fail as it's in the outer exception. But I've dealt with it.